README.md

```diff
@@ -1,22 +1,235 @@
+    │                                     command line will be used to create the  │
+    │                                     application.                             │
+    │                                     [default: None]                          │
+    │ --output_path         -o      TEXT  The path to the output directory where   │
+    │                                     the generated files will be saved. This  │
+    │                                     path should be relative to the current   │
+    │                                     working directory.                       │
+    │                                     [default: ./faststream-gen]              │
+    │ --verbose             -v            Enable verbose logging by setting the    │
+    │                                     logger level to INFO.                    │
+    │ --install-completion                Install completion for the current       │
+    │                                     shell.                                   │
+    │ --show-completion                   Show completion for the current shell,   │
+    │                                     to copy it or customize the              │
+    │                                     installation.                            │
+    │ --help                              Show this message and exit.              │
+    ╰──────────────────────────────────────────────────────────────────────────────╯
+
 ## How to use
 
-Fill me in please! Don’t forget code examples:
+This library is very easy to use. The only thing you need to do is
+prepare a description of your `FastStream` application, i.e. write what
+your application should do and pass it to the `faststrem_gen` command as
+a `string`:
+
+``` sh
+faststream_gen "your_app_description"
+```
+
+### Example
+
+``` sh
+faststream_gen  "Write a faststream application with with one consumer function and two producer functions. The consumer function should receive the a message posted on 'new_joinee' topic. The message should contain 'employee_name', 'age', 'location' and 'experience' attributes. After consuming the consumer function should send the details to the 'project_team' and 'admin_team' topics. Use only localhost broker" 
+✨  Generating a new FastStream application!
+ ✔ Application description validated 
+ ✔ FastStream app skeleton generated and saved at: ./faststream-gen/application_skeleton.py 
+ ✔ FastStream app generated and saved at: ./faststream-gen/application.py 
+ ✔ Tests are generated and saved at: ./faststream-gen/test.py 
+ Tokens used: 18572
+ Total Cost (USD): $0.05635
+✨  All files were successfully generated!
+```
+
+**Note**: If you have longer application description (and we encourage
+writing a detailed description), it is easier to save description into
+the file and use:
+
+``` sh
+faststream_gen -i path_to_app_description/description.txt
+```
+
+`faststream_gen` CLI will create `faststream-gen` folder and save
```

Comment:
Mention `faststream-gen` is the default path and the users can change it by passing --output_path option in CLI